static/app/components/events/interfaces/crashContent/stackTrace/rawContent.tsx

```diff
@@ -320,16 +320,44 @@ function getFrame(
   }
 }
 
-export default function displayRawContent(
-  data: StacktraceType | null,
-  platform?: string,
-  exception?: ExceptionValue,
+type DisplayRawContentArgs = {
+  data: StacktraceType | null;
+  exception?: ExceptionValue;
+  hasSimilarityEmbeddingsFeature?: boolean;
+  includeJSContext?: boolean;
+  includeLocation?: boolean;
+  newestFirst?: boolean;
+  platform?: string;
+  rawTrace?: boolean;
+};
+
+/**
+ * For the given stack trace, generates an array of platform-specific raw content (strings)
+ * representing the frames, with configurable display options.
+ *
+ * @param args                                The arguments object for this function.
+ * @param args.data                           The parsed stack trace data.
+ * @param args.platform                       The platform of this stack trace.
+ * @param args.exception                      The exception captured by this stack trace.
+ * @param args.hasSimilarityEmbeddingsFeature Whether the similarity embeddings feature is enabled.
+ * @param args.includeLocation                Whether to include location (e.g. line number, column number) in stack trace frames.
+ * @param args.rawTrace                       If true, the generated stack trace will be in the default format for the platform.
+ *                                            If false, the stack trace will be structured according to args.newestFirst.
+ * @param args.newestFirst                    Whether to display the frames from newest to oldest.
+ * @param args.includeJSContext               Whether to include source code context in stack trace frames for JavaScript.
+ *
+ * @returns Array of formatted strings representing the stack trace, one per frame.
+ */
```

Comment:
these might be more useful directly on the DisplayRawContentArgs type

Reply:
had actually considered that but figured it might be helpful for hover on `displayRawContent` to show the args - but agreed they should really be on the type. will fix